src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -389,18 +396,38 @@ class ContentOpenWith extends PureComponent<Props, State> {
                             integrations={integrations}
                         />
                     )}
-                    {shouldRenderIntegrationPortal && (
+                    {shouldRenderLoadingIntegrationPortal && (
```

Comment:
What are your thoughts on combining this render block to be `shouldRenderLoadingIntegrationPortal || shouldRenderErrorIntegrationPortal` to save on the duplication of the `IntegrationPortal` and containing div?

Reply:
Good point. I tried this and ran into some weird issues when trying to copy they styles. I'll try again and figure out what's actually going on.